src/main/resources/help/content/deployment.md

```diff
@@ -2,57 +2,57 @@
 
 ${solution_name} can be run in either of the following modes:
 
-1. Host mode, on a Linux machine (or Linux VM) with Docker.
-2. Container mode, inside a container running on an orchestration platform such as Kubernetes, OpenShift, etc.
+1. Host mode on a Linux machine or a Linux virtual machine (VM) with Docker.
+2. Container mode inside a container running on an orchestration platform such as Kubernetes, OpenShift, etc.
 
-Each mode is discussed in detail below.
+Each mode is discussed as follows.
 
 ### Important notes regarding deployment sample code
 
 The deployment samples provided are intended to illustrate possible approaches to the challenges
 involved in deploying ${solution_name}. They are not intended to be used as-is in production.
 You should understand the code before you use it. They do not represent the only way to deploy in each environment.
 
-Your deployment approach will be the same whether you are invoking ${solution_name} directly, or invoking it via Detect.
+Your deployment approach is the same whether you are invoking ${solution_name} directly, or invoking it using Detect.
 Most of the sample deployments use Detect simply because that is the most common use case.
 
-### Using host mode on Linux machine (or Linux VM) with Docker
+### Using host mode on Linux/Linux VM with Docker
```

Comment:
The important distinction here is running on a real machine or a VM (vs. a container). I feel like that distinction is less clear with this change. For example, I might read the new header and think the section applies to my container  since it's a Linux container, but it does not. (Whether it's a real machine, a VM, or a container, the OS must be Linux.)